proposal/calc-functions.md

```diff
@@ -0,0 +1,363 @@
+
+Remove the existing definition of `CalcExpression`.
+
+> Note: we aren't adding `| CssRound | CssAbs` to the definition of `CalcValue`
+> because existing Sass releases already allow the global `round()` and `abs()`
+> functions in calculations.
+
+### `CssRound`
+
+Add the following production:
+
+<x><pre>
+**CssRound** ::= 'round('¹ CalcArgument (',' CalcArgument){2} ')'
+</pre></x>
+
+1: The string `round(` is matched case-insensitively.
+
+> Although the three-argument `round()` function only allows a few values in its
+> first argument, for simplicity those are checked at evaluation time rather
+> than parse time.
+
+### `CssAbs`
+
+Add the following production:
+
+<x><pre>
+**CssAbs** ::= 'abs('¹ CalcArgument ')'
+</pre></x>
+
+1: The string `abs(` is matched case-insensitively.
+
+## Procedures
+
+### Simplifying a Calculation
+
+Replace [the definition of "Simplifying a Calculation"] with the following:
+
+[the definition of "Simplifying a Calculation"]: ../spec/types/calculation.md#simplifying-a-calculation
+
+This algorithm takes a calculation `calc` and returns a number or a calculation.
+
+> This algorithm is intended to return a value that's CSS-semantically identical
+> to the input.
+
+* If `calc` was parsed from an expression within a `SupportsDeclaration`'s
+  `Expression`, but outside any interpolation, return a `calc` as-is.
+
+* Let `arguments` be the result of [simplifying](#simplifying-a-calculationvalue) each
+  of `calc`'s arguments.
+
+* If `calc`'s name is `"calc"` and `arguments` contains exactly a single number
+  or calculation, return it.
+
+* If `calc`'s name is `"mod"`, `"rem"`, `"atan2"`, or `"pow"`; `arguments` has
+  fewer than three elements; and none of those are unquoted strings or
+  `CalculationInterpolation`s, throw an error.
+
+  > It's valid to write `pow(var(--two-args))` or `pow(#{"2, 3"})`, but
+  > otherwise calculations' arguments must match the expected number.
+
+* If `calc`'s name is `"sin"`, `"cos"`, `"tan"`, `"asin"`, `"acos"`, `"atan"`,
+  `"sqrt"`, `"abs"`, or `"log"` and `arguments` contains exactly a single number
+  with [known units], return the result of passing that number to the function
+  in [`sass:math`] whose name matches `calc`'s.
+
+  [known units]: #known-units
+  [`sass:math`]: ../spec/built-in-modules/math.md
+
+* If `calc`'s name is `"exp"` and `arguments` contains exactly a single number
+  `number` with [known units], return the result of calling `math.pow(math.$e,
+  number)`.
+
+* If `calc`'s name is `"sign"` and `arguments` contains exactly a single number
+  `number` with [known units]:
+
+  * If `number`'s value is positive, return `1`.
+  * If `number`'s value is negative, return `-1`.
+  * Otherwise, return a unitless number with the same value as `number`.
+  
+    > In this case, `number` is either positive zero, negative zero, or NaN.
+  
+  > To match CSS's behavior, these computations _don't_ use fuzzy comparisons.
```

Comment:
> Does "fuzzy comparisons" here mean this? https://github.com/sass/sass/blob/main/spec/types/number.md#fuzzy-equality

Yep

> Also, I'm slightly confused by what it means for this to "not use fuzzy comparisons" when determining whether a number is positive or negative

Fuzzy comparisons round numbers to the nearest 1e-11, so any positive double less than 1e-11 would be considered fuzzy-equal to 0 rather than positive.